### DISCUSSION TITLE
I cannot find Q&A mode

### DISCUSSION THREAD

**Discussion by kunlun122** [langgenius/dify#10126] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Refer to https://docs.dify.ai/getting-started/install-self-hosted/docker-compose. I have installed dify by docker-compose ,But I cannot not found Q&A mode in knowledge-base. 
version is 0.10.1
![企业微信截图_1730367969578](https://github.com/user-attachments/assets/ee32e973-8962-4b42-b66b-2b9f38c98457)

**Comment by kunlun122** [comment]
How can I use Q&A mode ?

**Comment by crazywoola** [comment]
Switch to the high quality.

**Comment by kunlun122** [comment]
I have alread solved, upgrade version to 0.10.2
